src/service/profile.js

```diff
@@ -97,6 +97,7 @@ export const setUnFollow = async (data) =>
         resolve(res.data);
       })
       .catch((err) => {
+        console.error(err.response);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `console.error(err.response)` is a good step towards better error logging. However, it's important to note that `err.response` might be undefined in some cases, which could lead to an additional error. It would be safer to check if `err.response` exists before logging it.
```diff
- console.error(err.response);
+ if (err.response) {
+   console.error(err.response);
+ }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->